addons/mail/models/models.py

```diff
@@ -278,6 +279,26 @@ def _get_default_activity_view(self):
         templates = E.templates(activity_box)
         return E.activity(templates, string=self._description)
 
+    @api.model
+    def _sort_activities_by_record(self, activities, module=False):
```

Comment:
Probably this should be on mail thread or more specifically mail activity mixin and not generic records that don't even have activities.